tests/foreman/api/test_multiple_paths.py

```diff
@@ -247,52 +205,31 @@ def test_post_status_code(self, entity_cls):
         @Assert: HTTP 201 is returned with an ``application/json`` content-type
 
         """
-        logger.debug('test_post_status_code arg: %s', entity_cls)
-        skip_if_sam(self, entity_cls)
-
-        # Libvirt compute resources suffer from BZ 1118015. However, partials
-        # cannot be compared for class identity and the class hierarchy needs
-        # fixing (SatelliteQE/nailgun#42), so we just comment it out above.
-        if entity_cls in BZ_1118015_ENTITIES and bz_bug_is_open(1118015):
-            self.skipTest('Bugzilla bug 1118015 is open.')
-
-        response = entity_cls().create_raw()
-        self.assertEqual(httplib.CREATED, response.status_code)
-        self.assertIn('application/json', response.headers['content-type'])
+        exclude_list = (
+            entities.TemplateKind,  # see comments in class definition
+        )
+        for entity_cls in set(valid_entities()) - set(exclude_list):
+            with self.subTest(entity_cls):
+                logger.debug('test_post_status_code arg: %s', entity_cls)
+                skip_if_sam(self, entity_cls)
+
+                # Libvirt compute resources suffer from BZ 1118015. However,
+                # partials cannot be compared for class identity and the class
+                # hierarchy needs fixing (SatelliteQE/nailgun#42), so we just
+                # comment it out above.
+                if (entity_cls in BZ_1118015_ENTITIES and
+                        bz_bug_is_open(1118015)):
+                    self.skipTest('Bugzilla bug 1118015 is open.')
```

Comment:
The rest of the test will be run normally. 
Demo test:
```py
    def test_subtest_demo(self):
        i = 0
        for item in range(4):
            with self.subTest(item):
                if item is not 1:
                    print 'Running test %d' % item
                    i += 1
                else:
                    unittest2.SkipTest('I cannot execute value == 1 ')
                    print 'Skipping test %d' % item
                    i += 1
        print 'Total number of tests executed is %d' % i
```

Output:
```sh
nosetests --nocapture tests/foreman/cli/test_org_dupe.py
Running test 0
Skipping test 1
Running test 2
Running test 3
Total number of tests executed is 4
.
----------------------------------------------------------------------
Ran 1 test in 0.001s

OK

# python -m unittest2 tests/foreman/cli/test_org_dupe.py
Running test 0
Skipping test 1
Running test 2
Running test 3
Total number of tests executed is 4
.
----------------------------------------------------------------------
Ran 1 test in 0.001s

OK
```

Obviously we need https://github.com/xmlrunner/unittest-xml-reporting/issues/85 to make jenkins aware of the skipped ones.